manifests/e/Enigma/VirtualBox/11.00/Enigma.VirtualBox.installer.yaml

```diff
@@ -0,0 +1,22 @@
+# Created with komac v2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Enigma.VirtualBox
+PackageVersion: '11.00'
+InstallerLocale: en-US
+InstallerType: inno
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ReleaseDate: 2024-08-26
+AppsAndFeaturesEntries:

```

Comment:
This should be removed as it exactly matches the publisher specified in the default locale file